playbook.md

```diff
@@ -1177,32 +1094,10 @@ approaches and ideas if we have them - including where there's a better
 solution, but [at a cost](#charging-for-ticket-work). We always give options
 when we can.
 
-If you are in any doubt as to whether the client has understood and approved the
-change that you are about to make, explain it to them and ask for their explicit
-approval. It's fine to do this on the phone if that is more convenient, but if
-you do, update the ticket with a comment what you said and what was agreed.
-
-#### Following up on bad client feedback
```

Comment:
For the record, could I ask why you're removing this section? Is the information outdated, is it something we're no longer keen to have on a public page, something else?

This information could go in the commit message (I can show you how to amend it if you need/want), as a record for future readers and contributors to the Playbook.